colorine.go

```diff
@@ -94,16 +116,21 @@ func (logger *Logger) Log(prefix, message string) {
 		}
 	}
 
+	out := logger.getOutput()
+
+	logger.mu.Lock()
+	defer logger.mu.Unlock()
```

Comment:
Do we really need this? As getOutput() is guarded by a mutex, I don't think we need a mutex on the caller side.